### PR TITLE
Some code to build upon

### PR DESCRIPTION
**Pull request by ghost**
lays the ground work for initalisation, graphics, state-logic and game loop

### CODE REVIEW

File: mysterious_moose/src/graphics.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                    log.debug("fg_colour: " + str(colour))
+                    bg_colour = element["bg"] if "bg" in element else None
+                    style = element["style"] if "style" in element else None
+                    rotation = element["rotation"] if "rotation" in element else int(0)
+
+                    if element["type"] == "normal":
+                        self.fonts[element["font"]].render_to(self.display, (element["x"], element["y"]),
+                                                              element["text"], style=style,
+                                                              fgcolor=colour, bgcolor=bg_colour,
+                                                              rotation=rotation, size=element["size"])
```
Comment (fiddlen) [inline mysterious_moose/src/graphics.py]: if element["type"] == "normal":
                        self.fonts[element["font"]].render_to(
                            self.display, 
                            (element["x"], element["y"]),
                            element["text"], 
                            style=style,
                            fgcolor=colour, 
                            bgcolor=bg_colour,
                            rotation=rotation, 
                            size=element["size"]
                        )

File: mysterious_moose/src/graphics.py [hunk 2]
```diff
@@ -17,7 +17,8 @@ def __init__(self):
         if pygame_initialisation == (8, 0):
             log.info("PyGame Initialised")
         else:
-            log.critical("PyGame did not start correctly, attempting to continue as it could be fine?")
+            log.critical("PyGame did not start correctly, "
+                         "attempting to continue as it could be fine?")
```
Comment (fiddlen) [inline mysterious_moose/src/graphics.py]:
```
log.critical(
    "PyGame did not start correctly, "
    "attempting to continue as it could be fine?"
)
```

Comment (xedre) [reply mysterious_moose/src/graphics.py]: This line would be too long without this

**Comment by xedre** [comment]
It should be improved now

**Comment by ghost** [comment]
It should me fixed in both places now

**Pull request merged**